docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -119,17 +130,17 @@ Examples
 
 .. code-block:: cppkokkos
```

Comment:
Please use the "Examples" heading (see PR#385). Also, please make sure the code is both current and is copy - paste-able compiling and running code (in Compiler Explorer, for instance).